.github/workflows/conformance.yml

```diff
@@ -0,0 +1,79 @@
+name: Manually run upstream conformance test
+
+on:
+  workflow_dispatch:
+    inputs:
+      antrea-version:
+        description: The Antrea version to test. It could be a SHA-1 value, a branch, or a tag (e.g. a7b012b, release-1.12, v1.12.0).
+        default: main
+        required: true
+      k8s-version:
+        description: K8s version (e.g. v1.27.1). Kind's default K8s version will be used if empty.
+        required: false
+      test-suite:
+        description: The test suite to run. Valid values include "whole-conformance", "conformance", "network-policy, "sig-network", and "all".
+        default: whole-conformance
+        required: true
+      always-upload-logs:
+        description: Always upload logs regardless of the test result.
+        type: boolean
+        default: false
+
+env:
+  KIND_VERSION: v0.18.0
```

Comment:
Do we need to use latest version? I saw kind released v0.19.0 two weeks ago.

Reply:
The PR has no dependency on v0.19.0 so I prefer to align with other workflows in this PR. We can bump kind version later for all workflows.